### PR TITLE
Bullseye

### LINKED ISSUE #414: [Bullseye] 'php7.4-smbclient' has no installation candidate
### Describe the bug

Attempt to install Nextcloud_ynh on the bullseye CI fails because php7.x-smbclient doesn't exist on bullseye

### Context

- Hardware: Yunohost CI (LXC)
- YunoHost version: 11.x
- I have access to my server: *Through SSH | through the webadmin | direct access via keyboard / screen | ...*
- Are you in a special context or did you perform some particular tweaking on your YunoHost instance?: it's on bullseye ¯\\\_(ツ)_/¯
- Using, or trying to install package version/branch: master

### Steps to reproduce

- Install app on bullseye

### Expected behavior

The app succesfully installs

### Logs

```
95365 INFO [##++++++++..........] > Configuring PHP-FPM...
104156 WARNING E: Packages need to be removed but remove is disabled.
105518 WARNING Package php7.4-smbclient is not available, but is referred to by another package.
105519 WARNING This may mean that the package is missing, has been obsoleted, or
105519 WARNING is only available from another source
105519 WARNING E: Package 'php7.4-smbclient' has no installation candidate
105543 WARNING Unable to install dependencies
```

### PR DESCRIPTION
**Pull request by yalh76**
## Problem

- *Fix #414*

## Solution

- *Remove php-smbclient*
- *Some other stuff*

## PR Status

- [x] Code finished and ready to be reviewed/tested
- [x] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by yalh76** [comment]
!testme

File: scripts/restore [hunk 1]
```diff
@@ -134,6 +132,10 @@ mkdir -p "$datadir"
 
 # Fix app ownerships & permissions
 chown -R $app: "$final_path" "$datadir"
+find $final_path/ -type f -print0 | xargs -0 chmod 0644
+find $final_path/ -type d -print0 | xargs -0 chmod 0755
+find $datadir/ -type f -print0 | xargs -0 chmod 0640
+find $datadir/ -type d -print0 | xargs -0 chmod 0750
```
Comment (zamentur) [inline scripts/restore:138]: I think this commands with large quantity of files could break the script cause too much paths given to chmod :/

**Comment by ericgaspar** [comment]
!testme

File: scripts/restore (hunk 1 above)
Comment (zamentur) [reply scripts/restore:138]: It was already in this package and has been changed by an easier command

Comment (zamentur) [reply scripts/restore:138]: It seems to work on sans-nuage... I will try on an other instance

File: manifest.json [hunk 2]
```diff
@@ -22,7 +22,7 @@
         "email": "pierre@kayou.io"
     },
     "requirements": {
-        "yunohost": ">= 4.2"
+        "yunohost": ">= 4.2.0"
```
Comment (zamentur) [inline manifest.json:25]:
```suggestion
        "yunohost": ">= 4.2.8.2"
```

**Comment by zamentur** [comment]
I just tried to upgrade with sans-nuage, the script works, but we have currently an out of memory allocation error about mysql (i have upgraded before into 4.2.8.2). The nextcloud page display only the error page.

I am searching the reason why it fails.

**Comment by zamentur** [comment]
```
sort_buffer_size = 256K
```
Pas suffisant sur une installation réel, j'ai passé à 4M sur sans-nuage.fr

File: scripts/restore (hunk 1 above)
Comment (zamentur) [reply scripts/restore:138]: It works on several instances around 100GB of files

**Review by zamentur — APPROVED** [review]
LGTM tested manually

**Pull request merged**